src/main/java/duke/command/CommandReminder.java

```diff
@@ -0,0 +1,40 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.worker.Ui;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Calendar;
+import java.util.Date;
+
+public class CommandReminder extends Command {
+    //private String[] reminders;
+    protected Date currentDate = Calendar.getInstance().getTime();
+
+    public CommandReminder() {
+        this.currentDate.setTime(0);
+    }
+
+    @Override
+    public void execute(TaskList taskList) {
+        try {
+            for (Task task : taskList.getList()) {
+                Date dateCopy = task.getDateTime();
```

Comment:
You should probably abstract this. I think the Prof mentioned that code quality includes at most 2-3 layers of indentation so do try and adhere to this even though it's kind of tedious to do so.